x-pack/test/send_search_to_background_integration/tests/apps/management/search_sessions/sessions_management.ts

```diff
@@ -131,14 +110,14 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         expectSnapshot(expiresColText).toMatchInline(`
           Array [
             "--",
-            "3 Dec, 2020, 00:00:00",
-            "4 Dec, 2020, 00:00:00",
-            "5 Dec, 2020, 00:00:00",
+            "25 Dec, 2020, 00:00:00",
+            "24 Dec, 2020, 00:00:00",
+            "23 Dec, 2020, 00:00:00",
```

Comment:
Does this change when you rerun the test?

Reply:
It was wrong because we added sorting by descending creation date order